lib/bolt/applicator.rb

```diff
@@ -42,7 +42,7 @@ def custom_facts_task
         path = File.join(libexec, 'custom_facts.rb')
         file = { 'name' => 'custom_facts.rb', 'path' => path }
         metadata = { 'supports_noop' => true, 'input_method' => 'stdin' }
```

Comment:
We should be able to add sensitive by hard-coding the parameters, same as you put into the metadata in apply_helpers.

Reply:
That doesn't work for `apply_catalog` because it calls the executor and transport directly instead of going through the `run_task()` puppet function.

Reply:
Oh and the `apply_prep()` function calls the executor directly when running `custom_facts`, so that one doesn't work either.

Reply:
Ah, we'd have to pull in the logic to wrap parameter input in Sensitive objects.